documentation/src/main/asciidoc/topics/con_session_externalization.adoc

```diff
@@ -0,0 +1,21 @@
+[id='session-externalization_{context}']
+= Session externalization
+
+{brandname} can provide external caches for applications built on Red Hat JBoss Enterprise Application Platform (EAP), Red Hat Build of Quarkus, Red Hat JBoss Web Server (JWS), and Spring.
+These external caches store HTTP sessions and other data independently of the application layer.
+
+.Session externalization
+//Community content
+ifdef::community[]
+image::session-externalization-ispn.png[This illustration depicts a {brandname} cluster that offloads session data from applications.]
+endif::community[]
+//Downstream content
+ifdef::downstream[]
+image::session-externalization.png[This illustration depicts a {brandname} cluster that offloads session data from applications.]
+endif::downstream[]
+
+Externalizing sessions to {brandname} gives you the following benefits:
+
+Elasticity:: Make applications stateless to add and remove nodes without the need for rebalancing operations as well as to avoid downtime during cluster upgrades.
+
+Smaller memory footprints:: Storing session data in external caches reduces memory pressure for applications, resulting in shorter and less frequent garbage collection (GC) runs.
```

Comment:
On the contrary, externalizing sessions will generally result in more frequent GC runs, since session attribute objects will be created and discarded per request (rather than retained in memory across requests).  This can be mitigated by leveraging a near cache, in which case the heap requirements are comparable.

Reply:
Hmm... Thanks @pferraro I actually found this in some old JDG docs as I was trying to address why you'd want to externalize sessions in the first place. What if we drop "resulting in shorter and less frequent garbage collection (GC) runs."?

Does "storing session data in external caches reduces memory pressure for applications" make sense? Maybe "Storing session data in external caches reduces the overall amount of memory that applications require" is best?